Documentation/rust/arch-support.rst

```diff
@@ -23,7 +23,8 @@ support corresponds to ``S`` values in the ``MAINTAINERS`` file.
      - None
    * - ``powerpc``
      - Maintained
-     - ``ppc64le`` only, ``RUST_OPT_LEVEL >= 2``
+     - ``ppc64le`` only
+     - ``RUST_OPT_LEVEL < 2`` requires ``CONFIG_THREAD_SHIFT=15``
```

Comment:
This would create a fifth column in the table, no?

Reply:
Oops. Actually it seems to break the table entirely. Will fix.